import-export-cli/impl/getOperationPolicies.go

```diff
@@ -0,0 +1,145 @@
+/*
+*  Copyright (c) WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+*
+*  WSO2 Inc. licenses this file to you under the Apache License,
+*  Version 2.0 (the "License"); you may not use this file except
+*  in compliance with the License.
+*  You may obtain a copy of the License at
+*
+*    http://www.apache.org/licenses/LICENSE-2.0
+*
+* Unless required by applicable law or agreed to in writing,
+* software distributed under the License is distributed on an
+* "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+* KIND, either express or implied.  See the License for the
+* specific language governing permissions and limitations
+* under the License.
+ */
+
+package impl
+
+import (
+	"encoding/json"
+	"fmt"
+	"io"
+	"os"
+	"text/template"
+
+	"github.com/go-resty/resty/v2"
+	"github.com/wso2/product-apim-tooling/import-export-cli/formatter"
+	"github.com/wso2/product-apim-tooling/import-export-cli/utils"
+)
+
+// TODO: Add Policy Version
+const (
+	apiPolicyUUIDHeader               = "ID"
+	apiPolicyNameHeader               = "NAME"
+	apiPolicyDisplayNameHeader        = "Display NAME"
```

Comment:
We we have uppercase lowercase words mixed here?

Reply:
These constants are not getting exported by another package. Do we have to make the first letter capital?

Reply:
I meant the "Display NAME". Shouldn't it be "DISPLAY NAME"?

Reply:
ah, yeah. will correct this